### ISSUE TITLE
Link to heading or block in current file doesn't work without repeating current file name

### ISSUE THREAD

**Issue by NomarCub** [TfTHacker/obsidian42-strange-new-worlds#60]
You have to specify the name of the current note for them to work, which is not necessary otherwise.

Example file called Note.md:

![Screenshot from 2022-12-07 20-35-09](https://user-images.githubusercontent.com/5298006/206279414-517375bc-b832-4d02-adfe-737facd626eb.png)

```md
# heading

[[#heading]]
[[Note#heading]]

block ^block

[[Note#^block]]
[[#^block]]
```

**Comment by DutchPete** [comment]
When I link from a canvas card to a heading in a note, that heading does not display the small link count square, even after closing and relaunching Obs.

If this is a different issue, please tell me and I will open a new issue.

**Comment by TfTHacker** [comment]
Well, this one was a pain! I think I have it fixed. It should be released soon with 1.2.0 update to SNW. Thank you for reporting the issue.
